jina/resources/helloworld.reduce.yml

```diff
@@ -20,6 +20,6 @@ requests:
           start: 0
           end: 50
           granularity_range: [0, 0]
-          adjacency_range: [0, 1]
+          adjacency_range: [0, 0]
```

Comment:
New ranges are so much better than the old ones. This slice says: get me the top 50 in the first dimension e.g. for the first 50 queries submitted. I like it.